src/commands/cmd_list.cc

```diff
@@ -154,6 +154,79 @@ class CommandRPop : public CommandPop {
   CommandRPop() : CommandPop(false) {}
 };
 
+class CommandLMPop : public Commander {
+ public:
+  /// format: LMPOP #numkeys key0 [key1 ...] <LEFT | RIGHT> [COUNT count]
+  Status Parse(const std::vector<std::string> &args) override {
+    CommandParser parser(args, 1);
+
+    auto num_keys = GET_OR_RET(parser.TakeInt<uint32_t>());
+    keys_.clear();
+    keys_.reserve(num_keys);
+    for (uint32_t i = 0; i < num_keys; ++i) {
+      keys_.emplace_back(GET_OR_RET(parser.TakeStr()));
+    }
+
+    auto left_or_right = util::ToLower(GET_OR_RET(parser.TakeStr()));
+    if (left_or_right == "left") {
+      left_ = true;
+    } else if (left_or_right == "right") {
+      left_ = false;
+    } else {
+      return {Status::RedisParseErr, errInvalidSyntax};
+    }
+
+    if (parser.Good()) {
+      if (parser.EatEqICase("count")) {
+        count_ = GET_OR_RET(parser.TakeInt<uint32_t>());
+      } else
```

Comment:
I'd use curly braces with the `else` part of the statement.

Reply:
Well, this code is refactored by the naughty clang-format with "./x.py format". I personally don't like this no-brace style as well. Will try to add it back, but if the clang-format still says no, I have to follow its instruction to pass the format checking.

Reply:
Done.